### PR TITLE
docs: fix div without close tag

### PR DESCRIPTION
**Pull request by Dunqing**
fix
image
image

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     document      |
| 🇨🇳 Chinese |     document      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: docs/react/migration-v5.zh-CN.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ title: 从 v4 到 v5
 ### 设计规范调整
 
 - 基础圆角调整，由统一的 `2px` 改为四级圆角，分别为 `2px` `4px` `6px` `8px`，分别应用于不同场景，比如默认尺寸的 Button 的圆角调整为了 `6px`。
-- 主色调整，由 <div style="display: inline-block; width: 16px; height: 16px; border-radius: 4px; background: #1890ff; vertical-align: text-bottom;" /> `#1890ff` 改为 <div style="display: inline-block; width: 16px; height: 16px; border-radius: 4px; background: #1677ff; vertical-align: text-bottom;" /> `#1677ff`。
+- 主色调整，由 <div style="display: inline-block; width: 16px; height: 16px; border-radius: 4px; background: #1890ff; vertical-align: text-bottom;" /> `#1890ff` 改为 <div style="display: inline-block; width: 16px; height: 16px; border-radius: 4px; background: #1677ff; vertical-align: text-bottom;" ></div> `#1677ff`。
```
Comment (yoyo837) [inline docs/react/migration-v5.zh-CN.md:17]: ### 
```suggestion
- 主色调整，由 <div style="display: inline-block; width: 16px; height: 16px; border-radius: 4px; background: #1890ff; vertical-align: text-bottom;"></div> `#1890ff` 改为 <div style="display: inline-block; width: 16px; height: 16px; border-radius: 4px; background: #1677ff; vertical-align: text-bottom;" ></div> `#1677ff`。
```

Comment (yoyo837) [reply docs/react/migration-v5.zh-CN.md:17]: 每句话是俩, 只改了一个

**Comment by afc163** [comment]
@PeachScript 自闭合标签不支持么？

**Comment by Dunqing** [comment]
表格多一列是 pre-commit 执行 pretty-quick 自动加上的

**Comment by Dunqing** [comment]
> 表格多一列是 pre-commit 执行 pretty-quick 自动加上的

`motionEaseOut` 多了一列导致的

**Comment by PeachScript** [comment]
> @PeachScript 自闭合标签不支持么？

HTML 5 规范里的单标签/自闭合标签是可以的，规范外的估计 unified.js 就忽略掉了

**Review by MadCcc — APPROVED** [review]

**Pull request merged**